qiskit_nature/second_q/mappers/qubit_mapper.py

```diff
@@ -147,18 +150,15 @@ def map(
 
         if issubclass(wrapped_type, SparseLabelOp):
             second_q_ops = [second_q_ops]
-            suppress_none = False
```

Comment:
Are we truely able to remove this? What if map a single operator which turns out not to commute? What will be returned then? If `None`, the return type hint needs to also include `| None`